core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -610,6 +610,14 @@ public enum CoreError implements ScalarDbError {
       "Invalid file extension: %s. Allowed extensions are: %s",
       "",
       ""),
+  DATA_LOADER_INVALID_COLUMN_KEY_PARSING_FAILED(
```

Comment:
The key name `DATA_LOADER_INVALID_COLUMN_KEY_PARSING_FAILED` and the error message seem a bit inconsistent? "PARSING_FAILED" sounds that a string doesn't align with the proper syntax/format.

Reply:
Updated and renamed.